components/admission/configs/base/sync-git-token.yaml

```diff
@@ -10,11 +10,10 @@ spec:
         any:
           - resources:
               kinds:
-                - Secret
-              names:
-                - flux-system
-              namespaces:
-                - flux-system
+                - v1/Namespace
+              selector:
+                matchLabels:
+                  toolkit.fluxcd.io/tenant: "apps"
```

Comment:
I don't think this is the right place to do this, apps is just one of many tenants. A better place for this would be a policy here: https://github.com/controlplaneio-fluxcd/d1-fleet/tree/main/tenants/apps/policies

Reply:
hmm, but we have the secret generation there, and the sync to add the `token` field should happen before that. Anyway this is while we wait for the fix, maybe we should just wait.